cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc

```diff
@@ -144,6 +144,8 @@ This table comprises all the execution plan operators ordered lexicographically.
 | <<query-plan-let-select-or-anti-semi-apply, LetSelectOrAntiSemiApply>>     | Performs a nested loop. Tests for the absence of a pattern predicate that is combined with other predicates. | | |
 | <<query-plan-let-semi-apply, LetSemiApply>>                                | Performs a nested loop. Tests for the presence of a pattern predicate in queries containing multiple pattern predicates. | | |
 | <<query-plan-limit, Limit>>                                                | Returns the first 'n' rows from the incoming input. | | |
+| <<query-plan-list-constraints, ListConstraints>>                           | List constraints with either verbose of brief output. | Y | |
+| <<query-plan-list-indexes, ListIndexes>>                                   | List indexes with either verbose of brief output. | Y | |
```

Comment:
Since the first column is the name of the operator, I think we should write `ShowConstraints` and `ShowIndexes` to match the execution plan that is printed out when you run EXPLAIN. The description could be either show or list, even if I prefer show slightly for consistency.

Reply:
The consistency is that docs use ´list` everywhere :P But I agree that I should update the operator name to the correct one